src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/controller/PractitionerCaseController.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.controller
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsallocations.domain.OffenderManagerDetails
+import uk.gov.justice.digital.hmpps.hmppsallocations.service.GetUnallocatedCaseService
+import uk.gov.justice.digital.hmpps.hmppsallocations.service.exception.EntityNotFoundException
+
+@RestController
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+class PractitionerCaseController(
+  private val getUnallocatedCaseService: GetUnallocatedCaseService
+) {
+  @Operation(summary = "Retrieve unallocated case for practitioner case")
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "OK"),
+      ApiResponse(responseCode = "404", description = "Result Not Found")
+    ]
+  )
+  @PreAuthorize("hasRole('ROLE_MANAGE_A_WORKFORCE_ALLOCATE')")
+  @GetMapping("/cases/unallocated/{crn}/convictions/{convictionId}/practitionerCase")
```

Comment:
we've used the term `offenderManager` in allocations and its called `personManager` in workload, can we use either one of those rather than introducing a new term?

Reply:
[That ship sailed in February](https://github.com/ministryofjustice/hmpps-allocations/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/domain/UnallocatedCaseConvictions.kt#L44)

 😄 But I take your point